azkaban-common/src/main/java/azkaban/metrics/CommonMetrics.java

```diff
@@ -34,6 +34,8 @@
   private final MetricsManager metricsManager;
   private Meter dbConnectionMeter;
   private Meter flowFailMeter;
+  private Meter dispatchFailMeter;
+  private Meter dispatchSuccessMeter;
```

Comment:
Aren't these metrics web server only? If so, should they be in azkaban.webapp.WebMetrics?